storage/client/blob/src/samples/java/blob/BasicExample.java

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package blob;
+
+import com.azure.core.util.configuration.ConfigurationManager;
+import com.azure.storage.blob.BlockBlobClient;
+import com.azure.storage.blob.ContainerClient;
+import com.azure.storage.blob.StorageClient;
+import com.azure.storage.blob.models.ListBlobsOptions;
+import com.azure.storage.common.credentials.SharedKeyCredential;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.nio.charset.Charset;
+import java.util.Locale;
+
+/**
+ * This example shows how to start using the Azure Storage Blob SDK for Java.
+ */
+public class BasicExample {
+
+    public static void main(String[] args) throws Exception {
+
+        // From the Azure portal, get your Storage account's name and account key.
+        String accountName = getAccountName();
+        String accountKey = getAccountKey();
+
+        // Use your Storage account's name and key to create a credential object; this is used to access your account.
+        SharedKeyCredential credential = new SharedKeyCredential(accountName, accountKey);
+
+        /**
```

Comment:
Should use multi-line comments here not JavaDoc comments. Same goes for the other places that do this.

Reply:
Done.